ccd/deployment.yaml

```diff
@@ -0,0 +1,825 @@
+  USER_PROFILE_HOST: 'http://ccd-user-profile-api'
+  IDAM_USER_URL: 'http://idam-api'
+  IDAM_S2S_URL: 'http://service-auth-provider-api'
+  REFORM_SERVICE_NAME: 'ccd-definition-store-api'
+  REFORM_TEAM: 'ccd'
+  REFORM_ENVIRONMENT: 'local'
+  APPINSIGHTS_INSTRUMENTATIONKEY: 'fake-key'
+---
+#
+# CCD-DATA-STORE-API CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ccd-data-store-api
+  namespace: ccd
+data:
+  JAVA_OPTS: '-Xms8m -Xmx256m -Djava.security.egd=file:/dev/./urandom -XX:MaxMetaspaceSize=128m'
+  DATA_STORE_DB_HOST: 'shared-database'
+  DATA_STORE_DB_PORT: '5432'
+  DATA_STORE_DB_USERNAME: 'ccd_data_store'
+  DATA_STORE_DB_PASSWORD: 'ccd_data_store'
+  DATA_STORE_IDAM_KEY: 'AAAAAAAAAAAAAAAA'
+  SPRING_DATASOURCE_URL: 'jdbc:postgresql://shared-database:5432/ccd_data_store?stringtype=unspecified'
+  DATA_STORE_TOKEN_SECRET: 'sdsadahjsadhgaskjhdhasghgkjasd'
+  DATA_STORE_S2S_AUTHORISED_SERVICES: 'bulk_scan_orchestrator,ccd_gw'
+  DEFINITION_STORE_HOST: 'http://ccd-definition-store-api'
+  USER_PROFILE_HOST: 'http://ccd-user-profile-api'
+  IDAM_USER_URL: 'http://idam-api'
+  IDAM_S2S_URL: 'http://service-auth-provider-api'
+  REFORM_SERVICE_NAME: 'ccd-data-store-api'
+  REFORM_TEAM: 'ccd'
+  REFORM_ENVIRONMENT: 'local'
+  APPINSIGHTS_INSTRUMENTATIONKEY: 'fake-key'
+  CCD_DM_DOMAIN: 'http://dm-store'
+---
+#
+# CCD-CASE-MANAGEMENT-WEB CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ccd-case-management-web
+  namespace: ccd
+data:
+  IDAM_LOGIN_URL: 'https://localhost:8000/login'
+  CCD_GATEWAY_BASE_URL: 'http://localhost:3453'
+  CCD_ACTIVITY_BASE_URL: 'http://localhost:3455'
+  DM_URL: 'http://localhost:3453/documents'
+  DM_URL_REMOTE: 'http://dm-store/documents'
+---
+#
+# CCD-API-GATEWAY CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ccd-api-gateway
+  namespace: ccd
+data:
+  IDAM_BASE_URL: 'http://idam-api'
+  IDAM_USER_URL: 'http://idam-api'
+  IDAM_LOGOUT_URL: 'https://localhost:8000/login/logout'
+  IDAM_OAUTH2_TOKEN_ENDPOINT: 'http://idam-api/oauth2/token'
+  IDAM_S2S_URL: 'http://service-auth-provider-api'
+  IDAM_SERVICE_KEY: 'AAAAAAAAAAAAAAAA'
+  IDAM_SERVICE_NAME: 'ccd_gw'
+  IDAM_OAUTH2_CLIENT_SECRET: '123456'
+  PROXY_AGGREGATED: 'http://ccd-data-store-api'
+  PROXY_DATA: 'http://ccd-data-store-api'
+  PROXY_DEFINITION_IMPORT: 'http://ccd-definition-store-api'
+  PROXY_DOCUMENT_MANAGEMENT: 'http://dm-store'
+---
+#
+# DM-STORE CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dm-store
+  namespace: ccd
+data:
+  JAVA_OPTS: '-Xms8m -Xmx256m -Djava.security.egd=file:/dev/./urandom -XX:MaxMetaspaceSize=128m'
+  SERVER_PORT: '4460'
+  SPRING_DATASOURCE_URL: 'jdbc:postgresql://shared-database:5432/evidence'
+  SPRING_DATASOURCE_USERNAME: 'evidence'
+  SPRING_DATASOURCE_PASSWORD: 'evidence'
+  IDAM_USER_BASE_URI: 'http://idam-api'
+  IDAM_S2S_BASE_URI: 'http://service-auth-provider-api'
+  S2S_NAMES_WHITELIST: 'ccd_gw,cmc,bulk_scan_orchestrator,ccd_data'
```

Comment:
extension point for teams to add their own services, ccd_gw and ccd_data always need to be there but teams will have other ones